base/src/main/java/org/aya/tyck/order/IncrementalTycker.java

```diff
@@ -7,20 +7,24 @@
 import org.aya.concrete.resolve.ResolveInfo;
 import org.aya.concrete.stmt.Decl;
 import org.aya.concrete.stmt.Stmt;
+import org.aya.util.MutableGraph;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * Incremental and non-stopping compiler for SCCs.
  *
+ * @param declUsage   transpose of {@link ResolveInfo#declGraph()}. Vertex should be tycked first.
```

Comment:
Did you mean "Vertex should be tycked first"? Like, which vertices should be tycked first?

Reply:
This is a usage graph: (vertex, w) means `w` depends on `vertex`.

Reply:
Can you write "For each element (vertex, w) in the graph, the vertex should be tycked first" instead? I think the original comment is pretty vague

Reply:
Good idea